.github/workflows/pipeline.yaml

```diff
@@ -37,4 +37,7 @@ jobs:
           pylint fondant --exit-zero
       - name: Test with pytest
         run: |
-          poetry run pytest tests
\ No newline at end of file
+          poetry run pytest tests
+      - name: Deploy documentation
```

Comment:
This will now run on every PR and overwrite the same version since there is only one.

I would propose to use ReadTheDocs instead, since it can build PRs and host multiple versions at the time.